ironfish/src/rpc/routes/wallet/types.ts

```diff
@@ -15,6 +15,7 @@ export type RpcAccountTransaction = {
   burnsCount: number
   expiration: number
   timestamp: number
+  assetBalanceDeltas: string
```

Comment:
I think instead of sending this back as JSON we should do `Array<{ assetId: string, delta: number }>`

Reply:
updated to `Array<{ assetId: string, delta: string }>`!